### PR TITLE
Add walkthrough in SearchPreview

### PR DESCRIPTION
**Pull request by jyash97**
Demo:
tutorial

### CODE REVIEW

File: components/SearchSandbox/index.js [hunk 1]
```diff
@@ -18,8 +19,75 @@ const wrapper = css`
 	padding: 15px;
 `;
 
+/* eslint-disable-next-line */
+injectGlobal`
+	.__floater__body>div>div{
```
Comment (metagrover) [inline components/SearchSandbox/index.js]: Linting issues

File: components/SearchSandbox/index.js [hunk 2]
```diff
@@ -18,8 +19,75 @@ const wrapper = css`
...
+		text-align:left !important;
+	}
+`;
+
 export const SandboxContext = React.createContext();
 
+const joyrideSteps = [
```
Comment (metagrover) [inline components/SearchSandbox/index.js]: We should move this to a separate file

Comment (jyash97) [reply components/SearchSandbox/index.js]: Sure 👍

**Comment by jyash97** [comment]
**New Changes**
- Change position of button to near `help` button
- added `tooltip` and `message` to show status to user
- Make Joyride component reusable - `Walkthrough` 
- Introduced localStorage to track is user has already completed tutorial
- `showWalkthrough` prop is used to show button, `case` Mappings we only show for `paid` user

**Demo**
new walkthrough

@siddharthlatest @metagrover

**Pull request merged**